shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -2430,7 +2430,11 @@ - (BOOL)isPresentingViewController {
   return self.presentedViewController != nil || self.isPresentingViewControllerAnimating;
 }
 
-- (flutter::PointerData)generatePointerDataAtLastMouseLocation API_AVAILABLE(ios(13.4)) {
+- (flutter::PointerData)updateMousePointerDataFrom:(UIGestureRecognizer*)gestureRecognizer
+    API_AVAILABLE(ios(13.4)) {
+  CGPoint location = [gestureRecognizer locationInView:self.view];
```

Comment:
Should this be checking `self.viewIfLoaded` and not updating the location if it's not yet loaded?

Reply:
I dont know the implications of this. there wasn't such a check before. at this point we have received an event from UIKit, the view must exist?

Reply:
I'm not sure the implications either, maybe there are none if the Flutter view is guaranteed to be loaded at this point. 🙂

Reply:
should be fine. this is only called in gesture callbacks, at which point the view must have already been on screen.